lace/test_geometry.py

```diff
@@ -187,3 +187,24 @@ def test_predict_body_units_cm(self):
         mesh = Mesh()
         mesh.v = np.array([[150, 100, 50], [-150, 50, 150]])
         self.assertEqual(mesh.predict_body_units(), 'cm')
+
+    def test_flip(self):
+        raw_box = Mesh(vc('/unittest/serialization/obj/test_box.obj'))
+        box = Mesh(v=raw_box.v, f=raw_box.f)
+        box.reset_normals()
+        original_v = box.v.copy()
+        original_vn = box.vn.copy()
+        original_f = box.f.copy()
+        box.flip(axis=0)
+        box.reset_normals()
+        self.assertEqual(box.f.shape, original_f.shape)
+        self.assertEqual(box.v.shape, original_v.shape)
+        for face, orig_face in zip(box.f, original_f):
+            self.assertNotEqual(list(face), list(orig_face))
```

Comment:
This strikes me as a weak assertion, but that could be because I am not entirely clear on why the two should not be equal. Is it because the order is inverted?

Any chance this could be changed into a `self.assertEqual(list(face), some_transformation(orig_face))`?